storage/integration.go

```diff
@@ -127,7 +125,6 @@ func (s *Storage) UpdateIntegration(integration *model.Integration) error {
 			instapaper_password=$7,
 			fever_enabled=$8,
 			fever_username=$9,
-			fever_password=$10,
```

Comment:
The placeholder should be updated as well. It is currently failing with this error:

```
store: unable to update integration row: pq: could not determine data type of parameter $10
```

Did you test your changes?

Reply:
Fixed. I'm running it on my server now, but missed to test updating the password. 😔